src/vorta/assets/exclusion_presets/browsers.json

```diff
@@ -0,0 +1,67 @@
+[
+    {
+        "name": "Chromium cache and config files",
+        "patterns":
+        [
+            "fm:/.config/chromium/*/Local Storage",
+            "fm:/.config/chromium/*/Session Storage",
+            "fm:/.config/chromium/*/Service Worker/CacheStorage",
+            "fm:/.config/chromium/*/Application Cache",
+            "fm:/.config/chromium/*/History Index *",
+            "fm:/snap/chromium/common/.cache",
+            "fm:/snap/chromium/*/.config/chromium/*/Service Worker/CacheStorage",
+            "fm:/snap/chromium/*/.local/share/"
+        ],
+        "tags":["application:chromium", "type:browser", "os:linux"],
+        "author": "Divi"
```

Comment:
Is the `author` key needed for anything?

Reply:
Nope, only to attribute the author.

Reply:
Does that make sense? Contributors might  modify presets in the future. Furthermore since the author isn't shown in the GUI.

Reply:
@m3nu I had the same thought regarding edits and revisions.